### PR TITLE
Use User.remoteMethod instead of loopbacks method

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
This is needed for loopback-connector-remote authorization.
Addresses https://github.com/strongloop/loopback/issues/622.

### CODE REVIEW

File: common/models/user.js [hunk 1]
```diff
@@ -319,7 +319,7 @@ module.exports = function(User) {
       options.port +
       options.restApiRoot +
       userModel.http.path +
-      userModel.confirm.http.path +
+      '/confirm' +
```
Comment (bajtos) [inline common/models/user.js]: This is changing configurable `confirm.http.path` to hard-coded `/confirm`, isn't that a breaking change?

Comment (BerkeleyTrue) [inline common/models/user.js]: Maybe, but there isn't a way that I am aware of to get the information dynamically. Using Model.remoteMethod does not make these properties available like loopback.remoteMethod does.

Comment (bajtos) [inline common/models/user.js]: > get the information dynamically. Using Model.remoteMethod does not make these properties available like loopback.remoteMethod does.

You need to call `sharedClass.find` to get the method metadata:

```
userModel.sharedClass.find('confirm', true).http.path
```

Comment (BerkeleyTrue) [inline common/models/user.js]: Sweet! Good to know. I'll make the changes tomorrow morning.

**Comment by ritch** [comment]
triggering the CI tests:

test please

**Comment by bajtos** [comment]
@slnode test please

@ritch IIRC Jenkins looks at the first line of the comment only

**Pull request merged**